app/workers/data_exporter.rb

```diff
@@ -2,16 +2,21 @@ class DataExporter
   include Sidekiq::Worker
 
   def perform(importer_class, data_export_id)
+    Rails.logger.info "Data export ##{data_export_id} — #{importer_class}: Sidekiq running. Loading DataExport record"
```

Comment:
Maybe picky but it feels wrong to use Sidekiq internals. I'm wondering if we can track the job without using the JID.

Perhaps could add structured logging in the job so you can look up Sidekiq logs by adding this at the start:

```
RequestLocals.store[:debugging_info] = { data_export_id: data_export_id, importer_class: importer_class }
```

This will be picked up by logstash: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/lib/logstash_logging.rb#L70-L72

Would that prevent us from needing the `sidekiq_jid`? It'll also remove the need for prefixing the log lines with "Data export ##{data_export_id} — #{importer_class}".

Reply:
Thank you, great — didn't know about `debugging _info`! I'll lose the JID reference. I guess we'll know by absence if the job was never enqueued.

Reply:
Actually I'm now wondering if we can just add the params to the log by default, so we'll have this for all Sidekiq logs. Some middleware?

Reply:
This is now ready to go I think — will return to that later if time